frontend/src/common/config/pagination.config.ts

```diff
@@ -0,0 +1,5 @@
+export const PaginationConfig = {
```

Comment:
Asta in ciuda numelul este o constanta si ar trebui sa fie in common/constants. Numele fisierului ar trebui sa fie pagniation.ts